src/components/numeral-date/numeral-date.test.tsx

```diff
@@ -0,0 +1,1995 @@
+  it("should render with `aria-invalid` set to 'false'", () => {
+    render(
+      <CarbonProvider validationRedesignOptIn>
+        <NumeralDate
+          value={{ dd: "02", mm: "01", yyyy: "2020" }}
+          onChange={() => {}}
+          info
+        />
+      </CarbonProvider>
+    );
+
+    const dayInput = screen.getByRole("textbox", { name: "Day" });
+    const monthInput = screen.getByRole("textbox", { name: "Month" });
+    const yearInput = screen.getByRole("textbox", { name: "Year" });
+
+    expect(dayInput).not.toBeInvalid();
+    expect(monthInput).not.toBeInvalid();
+    expect(yearInput).not.toBeInvalid();
+  });
+
+  it("should not display the tooltip when the user hovers on the 'Day' input", async () => {
+    const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+    render(
+      <CarbonProvider>
+        <NumeralDate
+          value={{ dd: "02", mm: "01", yyyy: "2020" }}
+          onChange={() => {}}
+          info
+        />
+      </CarbonProvider>
+    );
+
+    const dayInput = screen.getByRole("textbox", { name: "Day" });
+    await user.hover(dayInput);
+
+    expect(screen.queryByRole("tooltip")).not.toBeInTheDocument();
+  });
+
+  it("should not display the tooltip when the user hovers on the 'Month' input", async () => {
+    const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+    render(
+      <CarbonProvider>
+        <NumeralDate
+          value={{ dd: "02", mm: "01", yyyy: "2020" }}
+          onChange={() => {}}
+          info
+        />
+      </CarbonProvider>
+    );
+
+    const monthInput = screen.getByRole("textbox", { name: "Month" });
+    await user.hover(monthInput);
+
+    expect(screen.queryByRole("tooltip")).not.toBeInTheDocument();
+  });
+
+  it("should not display the tooltip when the user hovers on the 'Year' input", async () => {
+    const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
+    render(
+      <CarbonProvider>
+        <NumeralDate
+          value={{ dd: "02", mm: "01", yyyy: "2020" }}
+          onChange={() => {}}
+          info
+        />
+      </CarbonProvider>
+    );
+
+    const yearInput = screen.getByRole("textbox", { name: "Year" });
+    await user.hover(yearInput);
+
+    expect(screen.queryByRole("tooltip")).not.toBeInTheDocument();
+  });
+});
+
+test("should render the validation message when the `error` prop is passed a string value and `validationRedesignOptIn` is set", () => {
+  render(
+    <CarbonProvider validationRedesignOptIn>
+      <NumeralDate
+        value={{ dd: "02", mm: "01", yyyy: "2020" }}
+        onChange={() => {}}
+        error="error"
+      />
+    </CarbonProvider>
+  );
+
+  const dayInput = screen.getByRole("textbox", { name: "Day" });
+  const monthInput = screen.getByRole("textbox", { name: "Month" });
+  const yearInput = screen.getByRole("textbox", { name: "Year" });
+
+  expect(dayInput).toBeInvalid();
```

Comment:
**suggestion** (non-blocking): I'm wondering if you should make these "invalid" checks into separate tests for each of these cases, just as you did for the case where `validationRedesignOptIn` wasn't set. I don't actually mind much which way we do it (in theory I slightly prefer separate tests, but we already have a LOT of validation tests for this component...), but the inconsistency bothers me.

Reply:
I'm not convinced the minor discrepancy is a problem here tbh